### PR TITLE
Added error notifications to console

### LINKED ISSUE #99: Display Errors on UI
If an error is returned from the backend there should be a way to display them, for instance: 400, 401, 201, we can display as a snack bar or something:
proposal : https://material-ui.com/components/snackbars/
Screen Shot 2020-05-06 at 12 16 07 PM

### PR DESCRIPTION
**Pull request by bexsoft**
fixes #99 

## What does this do?

Implements new error snackbars in console and modals

## How does it look?

Screen Shot 2021-01-15 at 17 22 33

Screen Shot 2021-01-15 at 17 26 55

### CODE REVIEW

**Review by Alevsk — DISMISSED** [review]
LGTM

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx [hunk 1]
```diff
@@ -288,23 +293,23 @@ const ListObjects = ({
         xhr.status === 400 ||
         xhr.status === 500
       ) {
-        setSnackBarMessage(errorMessage);
+        setErrorSnackMessage("An error occurred while uploading the file.");
```
Comment (cesnietor) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx:296]: could you please set this to use `errorMessage`

**Review by cesnietor — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx:296]: that's my bad @bexsoft I picked the string when solving merge conficts

**Review by belvid — DISMISSED** [review]
Looks good to me

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx]: fixed, also changed the next two messages

**Review by cesnietor — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**